templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -27,6 +27,18 @@ contents:
         done
       fi
     }
+
+    # Used to replace an old master connection uuid with a new one on all connections
+    replace_connection_master() {
+      local old="$1"
+      local new="$2"
+      for conn_uuid in $(nmcli -g UUID connection show) ; do
+        if [ "$(nmcli -g connection.master connection show uuid "$conn_uuid")" != "$old" ]; then
```

Comment:
@jcaamano what if the old connection has a connection.master of device name instead of UUID, should we check that too?

Reply:
Done, we look for both things.